test/java/seedu/addressbook/commands/DeleteCommandTest.java

```diff
@@ -33,13 +33,13 @@
     @Before
     public void setUp() throws Exception {
         Person johnDoe = new Person(new Name("John Doe"), new Phone("61234567", false),
-                new Email("john@doe.com", false), new Address("395C Ben Road", false), new UniqueTagList());
+                new Email("john@doe.com", false), new Address("111, 395C Ben Road, #111, 111111", false), new UniqueTagList());
```

Comment:
Do note that if you are separating the parameters into multiple lines, the parameters should be aligned with each other. Sometimes the formatting style in IntelliJ is slightly different from what we expect.
e.g. 
```
Person johnDoe = new Person(new Name("John Doe"), 
                 new Phone("61234567", false),
                 ...
```

Please do make sure the line width is still not more than 110 chars